### ISSUE TITLE
Stop using abandoned flysystem-sftp -- there is flysystem-sftp-v3

### ISSUE THREAD

**Issue by kaurov** [BlackbitDigitalCommerce/pimcore-data-director#65]
- blackbit/data-director 2.8.8 requires league/flysystem-sftp * -> satisfiable by league/flysystem-sftp[2.5.0]. 
- blackbit/data-director 3.2.25 requires league/flysystem-sftp * -> satisfiable by league/flysystem-sftp[2.5.0].

It blocks me to update to phpseclib/phpseclib v.3 which is used in recent flysystem-bundle.

**Comment by BlackbitDevs** [comment]
Have used `league/flysystem-sftp-v3` in branch `3.3`. Am not sure whether it will work with Pimcore 6 with this though... we will see. But perhaps you can proceed with `3.3.x-dev` for now.

**Comment by BlackbitDevs** [comment]
Think in early january.

**Comment by kaurov** [comment]
Hello.
Thank you for your new version 3.2.30
I tested it.
But still it used old deprecated  league/flysystem-sftp 



composer require phpseclib/phpseclib:^3.0 --with-all-dependencies
./composer.json has been updated
Running composer update phpseclib/phpseclib --with-all-dependencies
Loading composer repositories with package information                                                      Updating dependencies                                 
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires phpseclib/phpseclib ^3.0, found phpseclib/phpseclib[3.0.0, ..., 3.0.18] but these were not loaded, likely because it conflicts with another require.
  Problem 2
    - league/flysystem-sftp 3.6.0 requires phpseclib/phpseclib ^2.0 -> found phpseclib/phpseclib[2.0.0, ..., 2.0.41] but it conflicts with your root composer.json require (^3.0).
    - blackbit/data-director 3.2.30 requires league/flysystem-sftp * -> satisfiable by league/flysystem-sftp[3.6.0].
    - blackbit/data-director is locked to version 3.2.30 and an update of this package was not requested.

**Comment by BlackbitDevs** [comment]
Yes, as I said, `league/flysystem-sftp-v3` is only on 3.3. Will release it in the next days.
You can already use it with the version constraint `3.3.x-dev` in your composer.json
